bugwarrior/notifications.py

```diff
@@ -91,6 +93,15 @@ def send_notification(issue, op, conf):
             priority=1,
         )
         return
+    elif notify_backend == 'applescript':
+        description = 'Finished querying for new issues.\n{}'.format(issue['description'])
+        notification = description if op == 'bw_finished' else message
+        subprocess.call([
+            'osascript',
+            '-e',
+            'display notification "{}" with title "Bugwarrior"'.format(notification)
```

Comment:
Should it escape? What happens if message is:  `Added task: Practice my "air quotes"`

then:

```shell
$ osascript -e 'display notification "Added task: Practice my "air quotes"" with title "Bugwarrior"'
46:50: syntax error: A identifier can’t go after this “"”. (-2740)
```

Reply:
`shlex.quote()` caused troubles when being passed into `osascript` on my end, since it always defaulted to single quotes...

```python
# works
subprocess.call(['osascript', '-e', 'display notification "{}" with title "bugwarrior"'.format('Test " Notification'.replace('"', '\\"'))])
# fails
subprocess.call(['osascript', '-e', 'display notification "{}" with title "bugwarrior"'.format(shlex.quote('Test " Notification'))])
```

Apologies if I'm missing something obvious again.